metricbeat/README.md

```diff
@@ -0,0 +1,19 @@
+These files is used to stream system metric.
```

Comment:
Please update readme to match that of collectd
- How to download
- Install
- Configure
- etc